### ISSUE TITLE
old overrides for UPS and canada post

### ISSUE THREAD

**Issue by mrpollo** [spree-contrib/spree_active_shipping#137]
these two classes where hot fixes for functionality not available inside active_shipping, these have fixed in active_shipping, so we need to either remove them or maintain them because shipping API's tend to change a lot frequently, I vote to remove them and let active_shipping deal with them since they have more contributors than we do

**Comment by jspizziri** [comment]
@mrpollo, I agree that these to classes should be removed. They have the potential to cause severe issues. I'm all in favor of eliminating them.

Can you tell me specifically what functionality they were intended to compensate for so that I can properly remove them?
